app/configurator/components/chart-options-selector.tsx

```diff
@@ -562,7 +566,63 @@ const EncodingOptionsPanel = (props: EncodingOptionsPanelProps) => {
         </ControlSection>
       )}
       {encoding.options?.showDots && (
-        <ChartShowDots fields={chartConfig.fields} field={field} />
+        <ControlSection collapse>
+          <SubsectionTitle iconName="chartLine">
+            <Trans id="controls.section.data-points">Data Points</Trans>
```

Comment:
@noahonyejese also here you shouldn't revert using `<ChartShowDots fields={chartConfig.fields} field={field} />`. I think something went wrong when rebasing this branch, I'd probably try to make it again to make sure we didn't miss anything else 👍